### ISSUE TITLE
Error!

### ISSUE THREAD

**Issue by Lucasbatt** [Jineapple/WololoKingdoms#16]
> There was an exception! Make sure you don't have the game running, or have any game files opened, then try running the installer again.
> If this still happens, please pass this error on to Jineapple: boost::filesystem::remove_all: Access is denied: "c:\program files (x86)\steam\SteamApps\common\Age2HD\Voobly Mods/AOC/Data Mods/WololoKingdoms FE\Data"
> 

Now trying a clean install. unistalled the game but i couldnt delete the folder inside "c:\program files (x86)\steam\SteamApps\common\Age2HD\Voobly Mods/AOC/Data Mods"

**Comment by Lucasbatt** [comment]
Now the steam installation isn't working

**Comment by Lucasbatt** [comment]
Uninstalling Voobly too now.

**Comment by Lucasbatt** [comment]
Reinstall all and: 

> There was an exception! Make sure you don't have the game running, or have any game files opened, then try running the installer again.
> If this still happens, please pass this error on to Jineapple: boost::filesystem::remove: The directory is not empty: "c:\program files (x86)\steam\SteamApps\common\Age2HD\Voobly Mods/AOC/Data Mods\WololoKingdoms\Script.Ai"

No. I don't have the game running

**Comment by Lucasbatt** [comment]
If anyone know anything that might work, just tell me. I'll try.
Or if anyone else is having the same issue.

**Comment by Jineapple** [comment]
Do you have any other programs that might be accessing those files? Something like a file sync, backup, voobly running? Apart from that, Maybe just try a few times, occasionally I see errors like that popping up but with no clear cause and running again usually fixes it. You could also try launching in safe mode and running the installer there if nothing else works.

**Comment by Lucasbatt** [comment]
It seems to launch the game when i press Run. 
Not sure if its supposed to happen.
https://youtu.be/-tni1VdmJ6w

**Comment by Lucasbatt** [comment]
Maybe it have something to do with "C:\Program Files (x86)\Steam\steamapps\common\Age2HD\Voobly Mods\AOC\Data Mods\Script.Ai\Brutal2" and "C:\Program Files (x86)\Steam\steamapps\common\Age2HD\Voobly Mods\AOC\Data Mods\WololoKingdoms\Script.Rm" since those are the only file I couldn't manually delete, access or move.

**Comment by Lucasbatt** [comment]
GOT IT!
I restarted several time to manage to delete those files. As there was no other file from the previous install (2.3) I ran the 2.5 wololo and everything when fine.
I guess win10 was scanning the files, so when I restarted I could delete them before win10 started scanning, allowing me to delete.
The Steam Client needs to be running and the Voobly client need to be closed. That last piece of information could be put on the README.txt to.
Thanks for the attention Jineapple. Also, I wouldn't have found where to report issues if it wasn't for the hint in the error message. Keep up the good work.
